cmd/lotus/daemon.go

```diff
@@ -174,6 +172,8 @@ var DaemonCmd = &cli.Command{
 			defer pprof.StopCPUProfile()
 		}
 
+		// FIXME: What is the difference between this profile option and
+		//  `cctx.Bool("bootstrap")`?
```

Comment:
Looking at this some more they do seem like they should be coupled. The bootstrap profile configures the PubSub peer manager (and the DHT as server) while the bootstrap CLI option turns off the the peer manager in the chain exchange service (and maybe others).

Reply:
```
NAME:
   lotus daemon - Start a lotus daemon process

OPTIONS:
   --bootstrap               (default: true)
   --profile value           specify type of node
```

To clarify, it seems `--profile` would mean "am _I_ a bootstrap node?" while `--bootstrap` is "should I be connecting to a bootstrap node?".

Reply:
https://github.com/filecoin-project/lotus/pull/5395